### DISCUSSION TITLE
Support customize checkout fields

### DISCUSSION THREAD

**Discussion by simonpham** [inspireui/support#5315] [Working on Features]
The app should allow customizing (add/remove) checkout fields

Or at least the webview checkout (mstore-checkout) should be compatible with this plugin:

[https://yithemes.com/themes/plugins/yith-woocommerce-checkout-manager/](https://yithemes.com/themes/plugins/yith-woocommerce-checkout-manager/)

**Comment by Emicloud91** [comment]
This would be great for custom checkouts and would make a BIG difference for many stores. :)

**Comment by regioNico** [comment]
I also have some custom checkout fields and these have to be pre-filled in the webview-checkout with values from the app. Right now I am not able to pass these values to the webview-checkout. Please provide a way for this too!

**Comment by skyedgedev** [comment]
Would be great to enable this feature. Would make a BIG difference.

**Comment by simonpham** [comment]
WIP. Will support hide/show/set default value/disable editing for currently supported fields.
Custom fields will be support in a later upgrade.

**Comment by HanielQC** [comment]
Hi guys, this option is available on FluxStore apps. More detail in https://www.notion.so/FluxStore-WooCommerce-591ca13557434bd88a9163e617d5eebd

Without purchase, you can view a demo App of your Website with the drag&drop FluxBuilder https://fluxbuilder.com/ for Free 🔻
🎯 Edit your App's UI by drag&drop
🎯 Build a Test APK to view how your App looks
Quick guide: https://www.notion.so/FluxBuilder-s-guide-a5164230ff444676b10b5cadc30302aa

Very Best!
